src/windows/wslaservice/exe/ContainerManager.h

```diff
@@ -0,0 +1,213 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+#include <string>
+#include <filesystem>
+#include <wil\resource.h>
+#include <map>
+#include <mutex>
+
+#include "WSLAVirtualMachine.h"
+
+namespace wsl::windows::service::wsla {
```

Comment:
In general we like our namespace names to match the file paths, so this would be wsl::windows::wslaservice